uint8 arm
---
bool result
